agent/command/shell.go

```diff
@@ -69,6 +72,11 @@ type shellExec struct {
 	// allows following commands to execute even if this shell command fails.
 	ContinueOnError bool `mapstructure:"continue_on_err"`
 
+	// Debug is a temporary and deliberately undocumented feature added in EVG-8017
+	// and should be removed. It writes a file called debug_shell_exec.<id>.yml that
```

Comment:
Are we ticketing this removal? Does STM only need this for some time?

Reply:
Opened https://jira.mongodb.org/browse/EVG-14311